<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox{
	font: 22px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	font:30px;
	color:#444444;
}</string>
    </property>
    <property name="text">
     <string>实验四：四则运算</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>251</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>四则运算</string>
    </property>
    <widget class="QDoubleSpinBox" name="data_A">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>70</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>运算参数A：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>运算参数B：</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="data_B">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>70</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>运算符号F：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="to_operate">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>140</y>
       <width>91</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>点击运算</string>
     </property>
    </widget>
    <widget class="QComboBox" name="data_F">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>100</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>+</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>*</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>/</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>90</y>
      <width>251</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>运算控制</string>
    </property>
    <widget class="QRadioButton" name="radio_zero">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>120</y>
       <width>131</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>整数部分为零</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">radio_group</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="radio_odd">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>131</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>整数部分为奇数</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">radio_group</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="radio_even">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>141</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>整数部分为偶数</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">radio_group</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>571</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>控制台</string>
    </property>
    <widget class="QTextBrowser" name="res_console">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>531</width>
       <height>91</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="radio_group"/>
 </buttongroups>
</ui>
